### ISSUE TITLE
Shapefile CRS conversion to EPSG:4326 - WGS 84 ?

### ISSUE THREAD

**Issue by Mike-Honey** [Code-for-All/geodata-scripts#6]
This is more of a question than an issue: is the python script already converting the shapefile CRS to EPSG:4326 - WGS 84 when it writes out .geojson files?

In my manual process using QGIS I have to Export a layer and choose that CRS to make the conversion. From my understanding, otherwise the geographic co-ordinates might be distorted.

**Comment by milovanderlinden** [comment]
You are right. I can see the source data is in EPSG:4674. I will make coordinate transformation a part of the process
